kurtosis.yml

```diff
@@ -1,4 +1,4 @@
-name: github.com/ethpandaops/optimism-package
```

Comment:
Please revert this change

Reply:
I wish this line was not needed as when you run a fork, it leads to an error and stops running